stake-pool/cli/src/main.rs

```diff
@@ -1444,6 +1419,7 @@ fn main() {
         ("withdraw", Some(arg_matches)) => {
             let pool_account: Pubkey = pubkey_of(arg_matches, "pool").unwrap();
             let burn_from: Pubkey = pubkey_of(arg_matches, "burn_from").unwrap();
+            // convert pool tokens
             let amount: u64 = sol_to_lamports(value_t_or_exit!(arg_matches, "amount", f64));
```

Comment:
Should this just be an amount instead of lamports since we're sending in pool tokens?

Reply:
Actually pool token has the same precision as SOL, so we still have to convert it. Probably need to set up a different function (like `pool_token_float_to_int`) or just keep `sol_to_lamports`?

Reply:
Ah I see, we can keep it then -- could you just add a comment explaining the same precision then?